packages/react/src/components/controls/StartAudio.tsx

```diff
@@ -5,35 +5,32 @@ import { useRoomContext } from '../../context';
 import { useObservableState } from '../../hooks/internal/useObservableState';
 import { mergeProps } from '../../utils';
 
-interface UseStartAudioProps {
-  room: Room;
-  props: React.ButtonHTMLAttributes<HTMLButtonElement>;
-}
-
-export function useStartAudio({ room, props }: UseStartAudioProps) {
-  const { className, roomAudioPlaybackAllowedObservable, handleStartAudioPlayback } = React.useMemo(
+/**
+ * In many browser to start audio playback, the user must perform a user-initiated event such as clicking a button.
+ * The `useStatAudio` hook returns an object with a boolean `canPlayAudio` flag
+ * that indicates whether audio playback is allowed in the current context,
+ * as well as a `startAudio` function that can be called in a button `onClick` callback to start audio playback in the current context.
+ *
+ * @see Autoplay policy on MDN web docs for more info: {@link https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API/Best_practices#autoplay_policy}
+ * @alpha
+ */
+export function useStartAudio(room: Room) {
```

Comment:
I also like `useAudioPlayback` as a more general term. the returned `canPlayAudio` makes more sense with that name imo

Reply:
`useAudioPlayback` is a better name, but we would break the naming convention of the component and the hook having the same name. I would then move the `useAudioPlayback` component to the hooks folder and remove it from here. What do you think about that?

Reply:
that sounds good! (but also seems like if we solve the issue above with two hooks, the naming problem would fix itself because we'd have a general hook anyways)